processing/src/main/java/io/druid/query/topn/AlphaNumericTopNMetricSpec.java

```diff
@@ -46,7 +46,7 @@ public AlphaNumericTopNMetricSpec(
   @Override
   public Comparator getComparator(List<AggregatorFactory> aggregatorSpecs, List<PostAggregator> postAggregatorSpecs)
   {
-    return new AlphanumComparator();
+    return ComparatorUtils.NATURAL_COMPARATOR;
```

Comment:
It seems it will break Alphanumeric sorting, 
e.g natural comparator sorts these in wrong order "NUM10071", "NUM9999"